packages/ssl/ssl.0.7.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for OpenSSL"
+maintainer: ["Antonio Monteiro <anmonteiro@gmail.com>"]
+authors: ["Samuel Mimram <samuel.mimram@ens-lyon.org>"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/savonet/ocaml-ssl"
+bug-reports: "https://github.com/savonet/ocaml-ssl/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.03.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0"}
```
due to Option being introduced in 4.08
```
### output ###
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I tests/.ssl_comm.eobjs/byte -I /home/opam/.opam/4.03/lib/alcotest -I /home/opam/.opam/4.03/lib/astring -I /home/opam/.opam/4.03/lib/cmdliner -I /home/opam/.opam/4.03/lib/fmt -I /home/opam/.opam/4.03/lib/re -I /home/opam/.opam/4.03/lib/seq -I /home/opam/.opam/4.03/lib/stdlib-shims -I /home/opam/.opam/4.03/lib/uuidm -I src/.ssl.objs/byte -no-alias-deps -o tests/.ssl_comm.eobjs/byte/dune__exe__Ssl_comm.cmo -c -impl tests/ssl_comm.ml)
# File "tests/ssl_comm.ml", line 13, characters 50-60:
# Error: Unbound module Option
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I tests/.util.objs/byte -I /home/opam/.opam/4.03/lib/alcotest -I /home/opam/.opam/4.03/lib/astring -I /home/opam/.opam/4.03/lib/cmdliner -I /home/opam/.opam/4.03/lib/fmt -I /home/opam/.opam/4.03/lib/ocaml/threads -I /home/opam/.opam/4.03/lib/re -I /home/opam/.opam/4.03/lib/seq -I /home/opam/.opam/4.03/lib/stdlib-shims -I /home/opam/.opam/4.03/lib/uuidm -I src/.ssl.objs/byte -no-alias-deps -o tests/.util.objs/byte/util.cmo -c -impl tests/util.ml)
# File "tests/util.ml", line 44, characters 8-21:
# Error: Unbound value List.find_opt
# Hint: Did you mean find_all?
```

Reply:
This should make the tests compile, leaving the package installable also on 4.03

Reply:
That was already there..?